modules/axhal/src/cpu.rs

```diff
@@ -22,6 +22,14 @@ pub fn this_cpu_is_bsp() -> bool {
     IS_BSP.read_current()
 }
 
+/// Gets the pointer to the current task without preemption-safety.
+///
+/// Preemption and irq must be disabled when calling this function.
+#[no_mangle]
```

Comment:
Remove `#[no_mangle]` and pass `xxx = sym xxx` in `global_asm!(include_str!(“trap.S”))`.